packages/core/apps/core-apps-server-internal/src/core_app.ts

```diff
@@ -144,6 +146,37 @@ export class CoreAppsService {
         }
       }
     );
+
+    router.versioned
+      .put({
+        path: '/internal/core/_settings',
+        access: 'internal',
+      })
```

Comment:
Should we add an access tag to the route? We could block its access to anyone but the superuser, as we're doing with the `_migrate` endpoint for the SO APIs:

https://github.com/elastic/kibana/blob/34c228b0c941308e01298e31223e7af0a5a2cee9/packages/core/saved-objects/core-saved-objects-server-internal/src/routes/migrate.ts#L21-L23

(any unknown `access` tag will automatically restrict to the SU only)